README.md

```diff
@@ -423,7 +423,7 @@ Usage:
 Flags:
       --erc20Address string   ERC20 contract address
   -h, --help                  help for add-minter
-      --minter string         address of minter
+      --handler string        handler contract address
```

Comment:
Arn't we decide to leave it as minter, but rather update the Readme, so users understand that in case of bridging role of minter should be given to the Handler?